types/src/legacy_types/cli_progress.rs

```diff
@@ -1,7 +1,9 @@
-//! standardize cli progress bars in 0L tools
```

Comment:
This is not a legacy type. It can be moved back to core_types

Reply:
Moved it to core_types.